packages-exp/auth-exp/src/model/public_types.ts

```diff
@@ -1190,3 +1190,14 @@ export interface Dependencies {
    */
   errorMap?: AuthErrorMap;
 }
+
+/**
+ * Custom token provider, which is invoked when a new Firebase ID token is requested and no refresh
+ * token is present (i.e., in passthrough mode).
```

Comment:
Mark this as `@public`:

```suggestion
 * token is present (i.e., in passthrough mode).
 *
 * @public
```

Reply:
Sick, thanks!